Thu Nov  3 18:55:37 2022
Windows-10-10.0.22000
Ren'Py 8.0.3.22090809
 

Bootstrap to the start of init.init took 0.04s
Early init took 0.00s
Loader init took 0.03s
Loading error handling took 0.03s
Loading script took 0.10s
Loading save slot metadata. took 0.02s
Loading persistent took 0.00s
Faled to initialize steam: FileNotFoundError("Could not find module 'C:\\Users\\seanc\\OneDrive\\Documents\\renpy-8.0.3-sdk\\lib\\py3-windows-x86_64\\steam_api64.dll' (or one of its dependencies). Try using the full path with constructor syntax.")
Running init code took 0.05s
Loading analysis data took 0.01s
Analyze and compile ATL took 0.00s
Index archives took 0.00s
Dump and make backups. took 0.04s
Cleaning cache took 0.00s
Making clean stores took 0.00s
Initial gc. took 0.02s
DPI scale factor: 1.500000
nvdrs: Loaded, about to disable thread optimizations.
nvdrs: b"Couldn't load nvlib." (can be ignored)
Creating interface object took 0.01s
Cleaning stores took 0.00s
Init translation took 0.04s
Build styles took 0.00s
Load screen analysis took 0.02s
Analyze screens took 0.00s
Save screen analysis took 0.00s
Prepare screens took 0.04s
Save pyanalysis. took 0.00s
Save bytecode. took 0.00s
Running _start took 0.00s
Performance test:
Interface start took 0.07s

Initializing gl2 renderer:
primary display bounds: (0, 0, 1920, 1080)
swap interval: 1 frames
Windowed mode.
Vendor: "b'Intel'"
Renderer: b'Intel(R) Iris(R) Xe Graphics'
Version: b'4.6.0 - Build 30.0.101.1273'
Display Info: None
Screen sizes: virtual=(1920, 1080) physical=(1731, 974) drawable=(1731, 974)
Maximum texture size: 4096x4096

Full traceback:
  File "game/script.rpy", line 1097, in script
    show makoto think right:
  File "C:\Users\seanc\OneDrive\Documents\renpy-8.0.3-sdk\renpy\ast.py", line 1408, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\seanc\OneDrive\Documents\renpy-8.0.3-sdk\renpy\ast.py", line 1371, in show_imspec
    renpy.config.show(name,
  File "C:\Users\seanc\OneDrive\Documents\renpy-8.0.3-sdk\renpy\exports.py", line 686, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\seanc\OneDrive\Documents\renpy-8.0.3-sdk\renpy\display\image.py", line 414, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\seanc\OneDrive\Documents\renpy-8.0.3-sdk\renpy\display\core.py", line 397, in _duplicate
    args.extraneous()
  File "C:\Users\seanc\OneDrive\Documents\renpy-8.0.3-sdk\renpy\display\core.py", line 265, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'makoto think' does not accept attributes 'right'.

While running game code:
  File "game/script.rpy", line 1097, in script
    show makoto think right:
Exception: Image 'makoto think' does not accept attributes 'right'.
